UHResidentInformationAPI/V1/Infrastructure/Address.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("property")]
+    public class Address
+    {
+       [Column("prop_ref")]
+        [MaxLength(9)]
+        [Key]
+        public int propertyRef { get; set; }
+
+        [Column("house_ref")]
+        [MaxLength(9)]
+        public int houseRef { get; set; } 
+
+         [Column("address1")]
```

Comment:
Might be some additional address-related columns needed here: https://app.swaggerhub.com/apis/Hackney/resident-housing-information-api/1.0.0#/address

Reply:
(Or I'll update the Swagger doc if the changes actually need to be made there.)